### PR TITLE
Cliqz logos available in Swift

### PR DESCRIPTION
**Pull request by chrmod**
Loading and updating the logo database has its cost, thus we should only do it once. 
This implementation gets the logos from browser-core. If it will prove itself non-performant, we should replace it with Swift based implementation. 

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] My code follows the code style of this project.
- [ ] I updated or created necessary unit tests
- [ ] I updated all necessary documentation

### CODE REVIEW

File: UserAgent/Home View/HomeViewController.swift [hunk 1]
```diff
@@ -25,6 +25,8 @@ class HomeViewController: UIViewController {
         case downloads
     }
 
+    private var logo = LogoView(url: "https://cliqz.com")
```
Comment (chrmod) [inline UserAgent/Home View/HomeViewController.swift]: This is just a demo code and should be remove when will implement History View.

File: ReactNative/Views/LogoView.swift [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                "url": url,
+                "size": self.bounds.width,
+            ]
+        )!
+
+        reactView.backgroundColor = .clear
+        return reactView
+    }()
+
+    override init(frame: CGRect) {
```
Comment (pavel-cliqz) [inline ReactNative/Views/LogoView.swift]: redundant

File: ReactNative/Views/LogoView.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        self.url = url
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("This class does not support NSCoding")
+    }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        self.addSubview(self.reactView)
```
Comment (pavel-cliqz) [inline ReactNative/Views/LogoView.swift]: in layoutSubviews self.addSubview(self.reactView) this is not nice. Even though the method is smart and won't add it multiple times, but it's not a good practice until you really have a good reason for this. If you simply want to add subview, do it in init of the view. layoutSubviews many many times, and here you need to only play with frames and constraints if required.

Comment (pavel-cliqz) [reply ReactNative/Views/LogoView.swift]: you can move this addSubview into override init(frame: CGRect)  method

**Pull request merged**